dcos.yml

```diff
@@ -10,38 +10,67 @@
         timeout: 120
 
 - name: Collect DC/OS versions
-  hosts: all
+  hosts: all!win_agents
+  gather_facts: no
   tasks:
     - name: Collect DC/OS versions
       setup:
 
+
 - name: DC/OS Requirements
-  hosts: all
+  hosts: all!win_agents
+  gather_facts: no
   become: true
   tasks:
     - include_role:
         name: DCOS.requirements
     - include_role:
         name: DCOS.gpu
 
-- name: "Setup and configure BOOTSTRAP nodes"
+
+- name: Setup and configure BOOTSTRAP nodes
   hosts: bootstraps
   become: true
   tasks:
     - include_role:
         name: DCOS.bootstrap
 
-- name: "Setup and configure MASTER nodes"
+
+- name: Setup and configure MASTER nodes
   hosts: masters
   serial: 1
   become: true
   tasks:
     - include_role:
         name: DCOS.master
 
-- name: "Setup and configure AGENT nodes"
+
+- name: Setup and configure AGENT nodes (Linux)
   hosts: agents
   become: true
   tasks:
     - include_role:
         name: DCOS.agent
+
+
```

Comment:
one newline is enough

Reply:
agreed, I used to do this from Python - 2 new lines